web/ui.js

```diff
@@ -739,7 +739,65 @@ $(function() {
                 var tradeCounts = data['trades'];
                 var useCounts = data['uses'];
 
-                var htmlData = "---------- <span title=\"Currency\">BTC</span> -----------\n";
+                var htmlData = "----- <span title=\"Total Portfolio\">TOTAL PORTFOLIO</span> -----\n";
+
+                // sum up altcoin totals
+                var altcoinTotals = {};
+                Object.keys(wallets).forEach(function(coin) {
+                    Object.keys(wallets[coin]).forEach(function(xid) {
+                        if (xid in altcoinTotals) {
```

Comment:
Once you address the previous comment, this part can be simplified...  This won't need to be an array any more.